diktat-rules/src/test/resources/test/paragraph3/string_concatenation/StringConcatenationExpected.kt

```diff
@@ -32,4 +32,5 @@ val myTest21 = "${x} string"
 val myTest22 = "string${foo()}"
 val myTest23 = x.toString() + foo()
 val myTest24 = foo() + "string"
-val myTest25 = "String ${valueStr?.value}"
\ No newline at end of file
+val myTest25 = "String ${valueStr?.value}"
```

Comment:
two issues contained 2 tests. Or they are the same?

Reply:
This is one problem caused by the lack of an IfExpression check.